package.json

```diff
@@ -38,42 +38,43 @@
     "deep-rename-keys": "^0.1.0",
     "del": "^2.2.0",
     "dot": "^1.0.3",
-    "dotenv": "^2.0.0",
-    "epochtalk-core-pg": "^1.0.8",
```

Comment:
this commit was also confusing because the commit title didn't label this commit specifically as a git merge.  will discuss this at git workflow meeting

Reply:
i missed it in the first commit.